README.md

```diff
@@ -260,6 +263,24 @@ BACKUP_CRON_EXPRESSION is a cron expression, in a Cron-Expression you define an
 Set BACKUP_CRON_EXPRESSION to change the default schedule.
 Example Usage: If BACKUP_CRON_EXPRESSION to `0 2 * * *`, the backup script will run every day at 2:00 AM.
 
+## Configuring Automatic Updates with Cron
+
+To be able to use automatic Updates with this Server the following environment variables **have** to be set to `true`:
+* RCON_ENABLED
+* UPDATE_ON_BOOT
```

Comment:
I would make it clear this only works if the container restarts.
You are stopping the game server so the container stops and so long as restart is enabled then it automatically updates otherwise you just stop the container.

It is possible to change it so the container says live and the game is updated. However that would require a pretty big change to either `init.sh` or `start.sh`.

Reply:
You're right I totally missed that. Thanks